### ISSUE TITLE
overfitting in training the yolov8-ImageNet-classifier

### ISSUE THREAD

**Issue by AlanLi1997** [ultralytics/ultralytics#980]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello, when I use the Yolov8 to train an ImageNet-classifier, I encountered the serious overfitting and a low accuracy:
![Q6%2 BL~T6HR$XVYEQU9SF8](https://user-images.githubusercontent.com/80144976/218932929-0ebbbb6f-2260-46ef-af40-1420300d3746.png)

Training command:

    yolo train classify data=imagenetmini model=yolov8n-cls.yaml dropout=0.2 epoch=600 batch=128 imgsz=224 patience=0...

the imagenetmini dataset at https://www.kaggle.com/datasets/ifigotin/imagenetmini-1000
Can I get these training-hyperparameters of the yolov8 used please? Or how can I solve the overfitting? (except using the dropout)

**Thanks!**

**Comment by AlanLi1997** [comment]
Hmm... It seems to be caused by too few samples for training... I changed the dataset and the overfitting was significantly dropped😅

**Comment by pderrenger** [comment]
@AlanLi1997 glad to hear that changing the dataset helped reduce the overfitting! 😊 Indeed, having an insufficient number of samples can lead to overfitting. It's great you were able to address the issue by adjusting the dataset. If you encounter any more questions or need further assistance, feel free to ask!
